src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -81,6 +81,21 @@ tenant_tunnel_network_vlanid=130
 tenant_vlan_range=201:250
 
 
+[MTU Settings]
```

Comment:
I think all of the variables below should be moved into the [Nodes Nics and Bonding Settings] section, and the *_network_mtu variables should be replaced with the actual variable names in the NIC config files.

Reply:
this was the decision made by sumedh and Randy to keep a separate section/stanza for mtu settings